enterprise/server/cmd/ci_runner/main.go

```diff
@@ -543,19 +553,21 @@ func (ar *actionRunner) Run(ctx context.Context, startTime time.Time) error {
 			Item: []*bespb.WorkspaceStatus_Item{
 				{Key: "BUILD_USER", Value: ar.username},
 				{Key: "BUILD_HOST", Value: ar.hostname},
-				{Key: "REPO_URL", Value: *repoURL},
+				{Key: "REPO_URL", Value: *pushedRepoURL},
 				{Key: "COMMIT_SHA", Value: *commitSHA},
-				{Key: "GIT_BRANCH", Value: *branch},
+				{Key: "GIT_BRANCH", Value: *pushedBranch},
 				{Key: "GIT_TREE_STATUS", Value: "Clean"},
-				// TODO: Consider parsing the `.bazelrc` and running the user's actual workspace
-				// status command that they've configured for bazel.
 			},
 		}},
 	}
 	if err := bep.Publish(workspaceStatusEvent); err != nil {
 		return nil
 	}
 
+	if ws.mergeConflict {
+		return fmt.Errorf("Merge conflict (merging branch %q into %q)", *pushedBranch, *targetBranch)
```

Comment:
looking at this code i see status errors returned -- is there a reason to return this as a fmt.Errorf?